docs/unified-test-documentation/dasharo-compatibility/320-update-firmware.md

```diff
@@ -0,0 +1,108 @@
+# Dasharo Compatibility: Update Firmware
+
+## Test cases
+
+### Common
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+1. Install the `dmidecode` package: `sudo apt install dmidecode`.
+
+### FFF001.001 Update firmware using downloaded files
+
+**Test description**
+
+This test verify whether it is possible to update the firmware on the DUT, and
+check the firmware version is correct.
+
+**Test configuration data**
+
+1. `FIRMWARE` = coreboot
+1. `OPERATING_SYSTEM` = Ubuntu 20.04
+
+**Test setup**
+
+1. Proceed with the [Common](#common) section.
+1. [Disable Secure Boot](../../unified-test-documentation/dasharo-security/206-secure-boot.md/#dasharo-security-uefi-secure-boot).
+
+**Test steps**
+
+1. Power on the DUT.
+
+1. Boot into the system.
+1. Log into system by using the proper login and password.
+1. Open a terminal window and run the follwing commands to download and extract
+   required files:
+    ```bash
+    sudo wget --content-disposition \
+     https://cloud.3mdeb.com/index.php/s/RHRtf72d6MoMdy2/download \
+     https://cloud.3mdeb.com/index.php/s/DqXD2So3eYrNf2y/download \
+     https://cloud.3mdeb.com/index.php/s/FqH7akAq6XPaweF/download \
+     https://cloud.3mdeb.com/index.php/s/GRA4e7J5pT4oSt4/download -P 
+     etc/fwupd/remotes.d
+    unzip fwupd-novacustom-v1.0.1.zip
+    unzip flashrom-1.2-2-3mdeb.zip
+    unzip fwupd-1.7.3.2-3mdeb.zip
+    ```
+1. Install requird packages:
+    ```bash
+    sudo apt install ./flashrom_1.2-2_amd64.deb \
```

Comment:
I think that instead of writing the whole command, we should put a link to it here. This allows us to avoid the situation when someone modifies one file and forgets to add such a modification to the second one.

Reply:
link aquired